src/HexagonOffload.cpp

```diff
@@ -12,13 +12,554 @@
 #include "Param.h"
 #include "RemoveTrivialForLoops.h"
 #include "Substitute.h"
+#include "Elf.h"
 
 namespace Halide {
 namespace Internal {
 
 using std::string;
 using std::vector;
 
+namespace Elf {
+
+enum {
```

Comment:
Is it worth adding a comment re: the source of these constants, for future reference?